README.md

```diff
@@ -117,7 +117,7 @@ See the [getting started](https://meshery.io/#getting-started) section to quickl
           </td>
         </tr>
         <tr>
-        <tr><td class="alpha-adapters"></td></tr>
+        <tr><td class="alpha-adapters"></td></tr></tr>
```

Comment:
The syntax here isn't valid. You'll want to use a <td></td> to achieve the desired affect instead.